### ISSUE TITLE
Suggestion: Fir Boats

### ISSUE THREAD

**Issue by Helpbot3000** [ProfessorProspector/Traverse#19]
I just think it would make sense.

**Comment by ProfessorProspector** [comment]
Yeah, I'd like to add one and I probably will eventually. It's just not a priority right now. There should be a crafting recipe to make an oak boat with fir planks which I added so you can at least make a boat with them.
